code/FBXDocument.h

```diff
@@ -654,6 +662,59 @@ typedef std::fbx_unordered_map<std::string, const Texture*> TextureMap;
 typedef std::fbx_unordered_map<std::string, const LayeredTexture*> LayeredTextureMap;
 
 
+/** DOM class for generic FBX videos */
+class Video : public Object
+{
+public:
+
+    Video(uint64_t id, const Element& element, const Document& doc, const std::string& name);
+    ~Video();
```

Comment:
I just did copy the Texture class above it, which neither has it, and wanted to keep it similar. In any case they are meant to be "final" classes, so virtual destructor is not essential.